config/locales/renewing_registrations.en.yml

```diff
@@ -58,8 +58,9 @@ en:
         dob: "Date of birth"
         convictions: "Convictions"
         has_convictions:
-          "yes": "Yes"
+          "YES": "Yes"
           "no": "No"
```

Comment:
```suggestion
          "NO": "No"
```

Reply:
Nope! :) We show `no` even when there is no result, so in the case of `no` it will be downcase. Only `YES` and `UNKNOWN` will happen to be called there

Reply:
I see now 🤦‍♀ Ignore me then!

Reply:
If you look at my commits, you will see I have made the same error and had to fix it!! :D